Autopilot/AttitudeManager/Network/Commands.h

```diff
@@ -74,6 +74,7 @@ typedef const enum {
     SET_ALTITUDE_GAINS = 143,
     SET_GROUND_SPEED_GAINS = 144,
     CALIBRATE_PWM_INPUTS = 145,
+    SET_PROBE = 146,
```

Comment:
This section is for Multi-part commands (commands with more than one data value). This should be a value less than 128.